src/layouts/initial-layout/InitialLayout.tsx

```diff
@@ -13,16 +19,38 @@ interface InitialLayoutProps {
 export const InitialLayout = ({ children }: InitialLayoutProps) => {
   const { t } = useTranslation('auth');
   const { pathname } = useLocation();
+  const { mode, setMode } = useConfigProvider();
+
+  useEffect(() => {
+    const handleKeyPress = (e: KeyboardEvent) => {
+      if (e.key === '|') {
```

Comment:
defined list keyPass on contant

Reply:
choose any keyPress you want. Don't need to define